packages/core/src/lib/connection_manager.ts

```diff
@@ -0,0 +1,381 @@
+import type { Connection } from "@libp2p/interface-connection";
+import type { PeerId } from "@libp2p/interface-peer-id";
+import type { PeerInfo } from "@libp2p/interface-peer-info";
+import type { Peer } from "@libp2p/interface-peer-store";
+import type { IRelay } from "@waku/interfaces";
+import { Tags } from "@waku/interfaces";
+import debug from "debug";
+import type { Libp2p } from "libp2p";
+
+import { KeepAliveManager, KeepAliveOptions } from "./keep_alive_manager.js";
+
+export const DEFAULT_MAX_BOOTSTRAP_PEERS_ALLOWED = 1;
+// TODO: add this functionality back
```

Comment:
What is the trigger for this TODO to be addressed in the future?

Reply:
Sure, I just wanted to understand what is blocking you to address it here.

Reply:
I didn't see it before and it is not clear these TODO's are connected. If they are maybe we can have an issue to track it and leave short TODO mentioning this issue?

Reply:
As discussed here: https://github.com/waku-org/js-waku/pull/1135#discussion_r1088554535 this TODO should be removed because this functionality is not needed.